content/admin/organization/orgs.md

```diff
@@ -104,6 +104,21 @@ configure your organization.
 - **Billing**: Displays information about your existing
 [Docker subscription (plan)](../../subscription/_index.md), including the number of seats and next payment due date. For how to access the billing history and payment methods for your organization, see [View billing history](../../billing/core-billing/history.md).
 
+## Merge organizations
+
+If you have multiple organizations that you want to merge into one, complete the following:
+
+1. Based on the number of seats from the secondary organization, [purchase additional seats](../../subscription/core-subscription/add-seats.md) for the primary organization account that you want to keep.
+2. Manually move users to the primary organization and remove existing users from the secondary organization.
```

Comment:
@kanwal-kaur Was curious if "move" or "add" is more accurate?

Reply:
For number 2, "Add" should be good. 
But let's keep "moving" in number 4

Thanks.